functional_requirements/requirements.txt

```diff
@@ -1,111 +1,188 @@
 The watch shall change option value or activate the selected option when button B is pressed. 
 
-	1) Brightness setting (default 50%)
-	2) Activate or deactivate bluetooth (default on)
-	3) If activated, news are notified and displayed in any view except hand clock view (default off)
-	4) If activated, a notification is sent when GPS position is more than 5km from configured home position (walk distance) (default off)
-	5) If activated, a notification is sent when when health status is considered problematic according to AI algorithm (default off)
-	6) Set the home position used in point 4 as the current GPS position. If GPS is not availble with enough precision (100 m), an error message is displayed.
-	7) Return to change view menu and save the current settings on storage card.
-			
-When a notification is sent a vibration is also sent through actuators (except for news notification).
+When "brightness" is selected, the user can select circularly between these possible values: 
+	Mid (50% - default) 
+	High (75%)
+	Low (25%)
+	Very High (100%)
 
-	
-Power management (background)
+when "bluetooth" is selected the user can choose to activate o deactivate the bluetooth.
+when "health status check" is selected the user can choose to activate o deactivate the health status check notification.
+when "return to change view menu" is selected, return to "Display Selection" menu and save the current settings.
+
+
+
+PART 3 Storage
+
+REQ 3.1 Storage card
+
+The watch shall provide a memory facility which will retain the information after power-off.
+
+REQ 3.2 Stored data 
+
+The permanent storage shall provide the possibility to store this information:
+
+	- Bluetooth enabled
+	- Brightness setting
+	- Health status check enabled
+
+REQ 3.3 Load data
+
+At power-on the watch shall read the stored data and apply them before enabling the interaction with the user.
+
+
+
+
+PART 4 Background management
+
+REQ 4.1 Battery reading
+
+The watch shall read the battery life status periodically
+
+REQ 4.2 Movement reading
+
+The watch shall monitor the accelerometer periodically to detect movement or lack of thereof.
+The movement is triggered when a force of X Newton is detected from the accelerometer (X is a calibration parameter).
+
+REQ 4.3 Battery failure
+
+If the battery life value is not available, a error is displayed to the user in Clock View.
+
+REQ 4.4 Low-power stand-by mode activation
+
+The watch shall go to low-power stand-by mode when a movement is not detected from accelerometer for at least 5 seconds or the power indicator is <=10% and >2%. 
+
+REQ 4.5 Low-power stand-by mode actions
+
+In Low-power stand-by mode the watch shall perfom these actions in order to reduce power consumption:
+
+	- Trigger motor vibration for 2 sec
+	- Set display brightness to "Low"
+	- Disable Bluetooth
+	- Disable Health status check notification
+	- Turn off display
+
+REQ 4.6 Normal mode activation

```

Comment:
State diagram and activity diagram that I have added call it "Full Power Mode". 

https://github.com/markroche92/sw-eng-smart-watch/blob/master/uml/state_diagrams/power_management.png
https://github.com/markroche92/sw-eng-smart-watch/pull/16/files?file-filters%5B%5D=.png&short_path=402469e#diff-402469e82262b4b52d8adc78faa9d9793b21844a95e2b88e4b0bdc5b71f33b3e

I can update to use call it Normal Mode if you like ?

Reply:
No, I fixed